### PR TITLE
Feature/ocbm 150 machine-info

### PR DESCRIPTION
**Pull request by JayshreeFlyers**
Implemented: 

- Created a constants file for the machine data. 
- Created to machine's data as objects.
- Added all sensor's data in constants.
- Added routing for the machine info page using id.
- Added the constants machine data for the machine info page.  

**Screenshots**
Screenshot 2023-11-16 150942
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Mastery from './pages/mastery';
 import Machines from './pages/machines';
 import 'react-toastify/dist/ReactToastify.css';
 import './App.css';
+import MachineInfo from './pages/machines/machineInfo';
```
Comment (harish-flyers) [inline src/App.tsx:15]: Keep import at top before the import of css to avoid order @JayshreeFlyers

File: src/assets/images/HMC1000largeimg.jpg
Comment (harish-flyers) [inline src/assets/images/HMC1000largeimg.jpg:1]: Pls follow standard for images filename eg:machine-HMC10000large.jpg  @JayshreeFlyers

**Review by harish-flyers — CHANGES_REQUESTED** [review]
Pls resolve comments @JayshreeFlyers

**Review by harish-flyers — COMMENTED** [review]
Looks good,
Better to avoid hardcode names for the images in alt attribute that too we can get from the utils machine data by keeping alt attribute there

**Comment by harish-flyers** [comment]
image
In breadcrumbs why for machine getting 1/2 instead of machine name? @JayshreeFlyers

File: src/assets/icons/index.tsx [hunk 2]
```diff
@@ -820,3 +820,12 @@ export const WithinSpecUnselected = ({ className }: IconsPropType) => {
     </svg>
   );
 };
+
+export const RunningIcon = () => {
```
Comment (chandran-kannan) [inline src/assets/icons/index.tsx:824]: add classname props

File: src/pages/machines/sensor.tsx [hunk 3]
```diff
@@ -1,32 +1,16 @@
 /* eslint-disable react/jsx-key */
```
Comment (chandran-kannan) [inline src/pages/machines/sensor.tsx:1]: fix this lint issue

File: src/pages/machines/sensor.tsx [hunk 4]
```diff
@@ -1,32 +1,16 @@
...
 const SensorPage = () => {
+  const navigate = useNavigate();
   return (
     <div className="flex gap-14 flex-wrap">
-      {sensors.map((sensorData) => (
+      {ALL_SENSOR_DATA.map((sensorData) => (
         <MachineCard
           key={sensorData.title}
-          handleView={() => ''}
+          handleView={() => navigate(sensorData.link)}
```
Comment (chandran-kannan) [inline src/pages/machines/sensor.tsx:13]: ensure this changes the route

**Review by chandran-kannan — CHANGES_REQUESTED** [review]

File: src/pages/machines/sensor.tsx (hunk 4 above)
Comment (JayshreeFlyers) [reply src/pages/machines/sensor.tsx:12]: yes bro. they have not given any specific screen to view specific sensor. In figma comments they say it should just show the graph page by clicking on it. so as of now i had made it like it navigates to the any sensor graph. if we click "view button".  @chandran-kannan

**Review by chandran-kannan — APPROVED** [review]

**Review by abineshflyers — APPROVED** [review]

**Review by harish-flyers — APPROVED** [review]

**Pull request merged**